install/common/synopsys-operator.yaml

```diff
@@ -82,6 +82,53 @@ items:
   metadata:
     name: synopsys-operator
     namespace: ${NAMESPACE}
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: synopsys-operator-admin
+  rules:
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources:
+    - customresourcedefinitions
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: ["security.openshift.io"]
+    resources:
+    - securitycontextconstraints
+    verbs: ["get", "update", "patch"]
+  - apiGroups: ["route.openshift.io"]
+    resources:
+    - routes
+    verbs: ["get", "create"]
+  - apiGroups: ["rbac.authorization.k8s.io"]
+    resources:
+    - clusterrolebindings
+    - clusterroles
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: [""]
+    resources:
+    - namespaces
+    - configmaps
+    - persistentvolumeclaims
+    - services
+    - secrets
+    - replicationcontrollers
+    - deployments
```

Comment:
Shouldn't deployment and statefulsets be in the **apps** apiGroups? Depending on how we use the deployment API, we may need to add it in the **extensions** apiGroups too.